## MAGI

MAGI is a tool for annotating, exploring, and analyzing gene sets that may be associated with cancer.
View aberrations in and interactions among gene sets.
Upload your mutation data
View in combination with TCGA Pan-Cancer data.
Generate dataset summaries.
Collaboratively annotate aberrations and interactions while exploring your data.
View the aberrations in a tumor sample, prioritized by their annotations.


http://magi.brown.edu/


##OASIS

About OASIS
OASIS is an open-access web portal that enables cancer researchers to perform exploratory and integrative analyses of somatic mutation, copy number changes (CNV) and gene expression data from thousands of tumor, normal tissues and cell lines representing a broad spectrum of malignancies.

OASIS is developed by Pfizer Oncology Research Computational Biology in collaboration with Research Business Technology (RBT). Please cite "OASIS: A web-based platform for exploratory analysis of cancer genome and transcriptome data (manuscript in preparation)" when publishing results based on OASIS.

The OASIS Team
Julio Fernandez-Banet
Anthony Esposito
Scott Coffin
Istvan Boerner-Horvath
Sabine Schefzick
Peter Roberts
Zhengyan 'George' Kan
Datasets
TCGA:
The Cancer Genome Atlas (TCGA) project is a joint effort of the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) to accelerate our understanding of the molecular basis of cancer through the application of genome analysis technologies, including large-scale genome sequencing. Please follow TCGA publication guidelines when using TCGA data in your publications.

CCLE1:
The Cancer Cell Line Encyclopedia (CCLE) project is a collaboration between the Broad Institute, and the Novartis Institutes for Biomedical Research and its Genomics Institute of the Novartis Research Foundation to conduct a detailed genetic and pharmacologic characterization of a large panel of human cancer models. The CCLE provides public access to genomic data for about 1000 cell lines. Please follow CCLE Terms of Usage when using CCLE data in your research.

GTEx2:
The Genotype-Tissue Expression (GTEx) project aims to provide a resource with which to study human gene expression and regulation and its relationship to genetic variation. This project will collect and analyze multiple human tissues from donors who are also densely genotyped, to assess genetic variation within their genomes.

ACRG-HCC3:
The ACRG-HCC dataset from the Asian Cancer Research Group contains array-based gene expression data and copy number variation, somatic mutation data based on whole genome sequencing (WGS) of a predominantly HBV driven Hepatocellular carcinoma (HCC) cohort.

Samsung-HCC4:
The Samsung-HCC dataset was generated through collaboration between Pfizer Oncology Research and the Samsung Medical Center (SMC) in South Korea. It contains array-based gene expression and copy number variation data from a HCC cohort.

HKU-GC5:
The HKU-GC dataset was generated through collaboration between Pfizer Oncology Research and the University of Hong Kong. It contains array-based gene expression, copy number variation data as well as mutation data based on whole exome sequencing (WES) and whole genome sequencing (WGS) on a cohort of gastric cancer patients.

METABRIC6:
The METABRIC dataset contains clinical traits, expression, CNV profiles, and SNP genotypes derived from breast tumors collected from participants of the METABRIC trial.

Acknowledgements
BioMart7:
The OASIS web portal was developed based on a custom version of the BioMart framework designed for oncogenomics data analysis8. The BioMart project provides free software and data services to the international scientific community in order to foster scientific collaboration and facilitate the scientific discovery process. We sincerely thank Arek Kasprzyk and the ICGC data portal development team for help with implementation of the BioMart framework.

OncoLand:
The Cancer Genome Atlas (TCGA) and the Genotype Tissue Expression (GTEx) analyses for this portal were generated using the OncoLand data service provided by Omicsoft Corporation. OncoLand, Array Studio, Array Viewer and Array Server and all other Omicsoft products or service names are registered trademarks or trademarks of Omicsoft Corporation, Cary, NC, USA.

cBio portal for Cancer Genomics9:
The original version of the Oncoprint code was provided by the cBio Portal for Cancer Genomics group at the Memorial Sloan Kettering Cancer Center.

Pfizer colleagues:
We would like to acknowledge the contributions of former team members Heather Estrella, Sarathy Mattaparti, Ming Cui and Yu 'David' Liu. We sincerely thank support and contributions from colleagues in Pfizer Oncology Research Computational Biology and Research Business Technology Kai Wang, Keith Ching, Shibing Deng, Ying Ding, Tao Xie, Zhou Zhu, Jack Yee, Tse Da, Susan Stephens and Paul Rejto.

References
1.	Barretina, J. et al. The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. Nature 483, 603-7 (2012). 
2.	Consortium, G.T. The Genotype-Tissue Expression (GTEx) project. Nat Genet 45, 580-5 (2013). 
3.	Kan, Z. et al. Whole-genome sequencing identifies recurrent mutations in hepatocellular carcinoma. Genome Res 23, 1422-33 (2013). 
4.	Wang, K. et al. Genomic landscape of copy number aberrations enables the identification of oncogenic drivers in hepatocellular carcinoma. Hepatology 58, 706-17 (2013). 
5.	Wang, K. et al. Whole-genome sequencing and comprehensive molecular profiling identify new driver mutations in gastric cancer. Nat Genet 46, 573-82 (2014). 
6.	Curtis, C. et al. The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Nature 486, 346-52 (2012). 
7.	Kasprzyk, A. BioMart: driving a paradigm change in biological data management. Database (Oxford) 2011, bar049 (2011). 
8.	Zhang, J. et al. International Cancer Genome Consortium Data Portal--a one-stop shop for cancer genomics data. Database (Oxford) 2011, bar026 (2011). 
9.	Cerami, E. et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov 2, 401-4 (2012).


http://oasis-genomics.org/

## cBioPortal

The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets.

Please adhere to the TCGA publication guidelines when using TCGA data in your publications.

Please cite Gao et al. Sci. Signal. 2013 & Cerami et al. Cancer Discov. 2012 when publishing results based on cBioPortal.

http://www.cbioportal.org/

## intOGen


The Biomedical Genomics Group

BG Group is a research group integrated in the Research Unit on Biomedical Informatics (GRIB) of the University Pompeu Fabra (UPF). The lab is located in the Biomedical Research Park in Barcelona (PRBB).

Our main research interest is the computational study of cancer from a genomics perspective (see details in Projects).
Citation

Rubio-Perez, C., Tamborero, D., Schroeder, MP., Antol√≠n, AA., Deu-Pons,J., Perez-Llamas, C., Mestres, J., Gonzalez-Perez, A., Lopez-Bigas, N. In silico prescription of anticancer drugs to cohorts of 28 tumor types reveals novel targeting opportunities. Cancer Cell 27 (2015), pp. 382-396

Gonzalez-Perez A, Perez-Llamas C, Deu-Pons J, Tamborero D, Schroeder MP, Jene-Sanz A, Santos A & Lopez-Bigas N IntOGen-mutations identifies cancer drivers across tumor types Nature Methods 2013; doi:10.1038/nmeth.2642
Tools and Data
IntOGen mutations uses several data sources and tools.
Tumor mutation data

Most tumour somatic mutations data (although not all) comes from: ICGC: The International Cancer Genome Consortium, TCGA: The Cancer Genome Atlas and independent projects. See in each project details for its source
Driver identification methods

OncodriveFM: Identifies genes with a bias towards high functional mutations, OncodriveCLUST: Identifies genes with a significant regional clustering of mutations MutSigCV: Identifies genes mutated more frequently than background mutation rate OncodriveROLE: Classifies driver genes according to its mode of action in Activation or Loss of Function
Mutations mapping

Ensembl VEP from Ensembl 70 has been used for all gene & transcript mapping as well as the official gene and transcript names. PFAM 27 domains have been downloaded from their ftp server and the obtained regions have been mapped using PRA-Tool.
Visualization

HighCharts has been used for various plots and Mutations Needle Plot library has been for the mutation distribution plots


https://www.intogen.org/search
